src/admin/redux/modules/email/reducer.js

```diff
@@ -95,6 +97,8 @@ const reducer : Reducer<State, Action>
 
 export default reducer
 
+const sort_by_date = r_compose(reverse, sortBy(prop('created_at')), map(over(lensProp('created_at'), Date.parse)))
```

Comment:
(There's also a similar function in `src/admin/redux/modules/payments`, but I think it's OK to have both)